sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/CreateNewEventSteps.java

```diff
@@ -41,10 +46,146 @@ public CreateNewEventSteps(WebDriverHelpers webDriverHelpers) {
+          selectEventStatus("SCREENING");
+          selectEventStatus("CLUSTER");
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(PRIMARY_MODE_OF_TRANSMISSION);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(NOSOCOMIAL);
+          selectEventStatus("DROPPED");
+          selectEventManagementStatusOption("PENDING");
+          selectEventManagementStatusOption("ONGOING");
+          selectEventManagementStatusOption("DONE");
+          selectEventManagementStatusOption("CLOSED");
+          selectRiskLevel("Low risk");
+          selectRiskLevel("Moderate risk");
+          selectRiskLevel("High risk");
+          selectRiskLevel("Unknown");
+          webDriverHelpers.scrollToElement(DISEASE_INPUT);
+          selectEventInvestigationStatusOptions("INVESTIGATION PENDING");
+          selectEventInvestigationStatusOptions("ONGOING INVESTIGATION");
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EVENT_INVESTIGATION_START_DATE);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EVENT_INVESTIGATION_END_DATE);
+          selectEventInvestigationStatusOptions("INVESTIGATION DONE");
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EVENT_INVESTIGATION_START_DATE);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EVENT_INVESTIGATION_END_DATE);
+          selectEventInvestigationStatusOptions("INVESTIGATION DISCARDED");
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EVENT_INVESTIGATION_START_DATE);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EVENT_INVESTIGATION_END_DATE);
+          selectDisease("Acute Flaccid Paralysis");
+          selectDisease("Anthrax");
+          selectDisease("COVID-19");
+          selectDisease("Cholera");
+          selectDisease("Congenital Rubella");
+          selectDisease("Dengue Fever");
+          selectDisease("Ebola Virus Disease");
+          selectDisease("Guinea Worm");
+          selectDisease("Human Rabies");
+          selectDisease("Influenza (New subtype)");
+          selectDisease("Lassa");
+          selectDisease("Measles");
+          selectDisease("Meningitis (CSM)");
+          selectDisease("Monkeypox");
+          selectDisease("Not Yet Defined");
+          selectDisease("Other Epidemic Disease");
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(DISEASE_NAME);
+          selectDisease("Plague");
+          selectDisease("Poliomyelitis");
+          selectDisease("Unspecified VHF");
+          selectDisease("Yellow Fever");
+          selectSourceType("Not applicable");
+          selectSourceType("Mathematical model");
+          selectSourceType("Media/News");
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(SOURCE_MEDIA_WEBSITE);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(SOURCE_MEDIA_NAME);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(SOURCE_MEDIA_DETAILS);
```

Comment:
do we need to wait for each one? I was thinking that the last visible one can cover all of them

Reply:
this is more like a validation when Media/News is selected only then these input fields should be there

Reply:
since its about validating all the fields that appear on different selected options we have to check all